site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
+title: Configure Application Email Templates
+description: I want to use application specific email templates to override Tenant level templates
+navcategory: developer
+---
+
+:page-liquid:
+
+:sectnumlevels: 0
+
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
```

Comment:
Add a table of contents.

Reply:
I use this command: `egrep '^[=]+ ' site/docs/v1/tech/PATH/TO/DOC |sed 's/=//' |sed 's/=/*/g'|sed 's/* /* <</'|sed 's/$/>>/'`